src/mono/msbuild/android/build/AndroidApp.targets

```diff
@@ -206,6 +217,7 @@ mono_thread_interruption_request_flag;" />
         AotModulesTablePath="$(_AotModuleTablePath)"
         Assemblies="@(_AotInputAssemblies)"
         CompilerBinaryPath="@(MonoAotCrossCompiler->WithMetadataValue('RuntimeIdentifier','$(TargetOS.ToLowerInvariant())-$(TargetArchitecture.ToLowerInvariant())'))"
+        DirectPInvokes="@(_DirectPInvokeLibs)"
```

Comment:
If we want to allow users of LibraryBuilder to specify their own custom managed functions to be DirectPInvokes that are directly callable, should we just make this `DirectPInvokes="@(DirectPInvokes)"`.
We can add `DirectPInvokeLists="@(DirectPInvokeLists)"` too while we are at it.